include/at_cmd_parser/at_params.h

```diff
@@ -32,13 +32,15 @@ extern "C" {
 /** Parameter types. */
 enum at_param_type {
 	/** Empty parameter. Must be value '0'. */
-	AT_PARAM_TYPE_EMPTY,
+	AT_PARAM_TYPE_INVALID,
 	/** Parameter of type short. **/
 	AT_PARAM_TYPE_NUM_SHORT,
 	/** Parameter of type integer. **/
 	AT_PARAM_TYPE_NUM_INT,
 	/** Parameter of type string. **/
-	AT_PARAM_TYPE_STRING
+	AT_PARAM_TYPE_STRING,
+	/** Parameter of type empty, not in use **/
```

Comment:
Maybe something like this?
```suggestion
	/** Emtpy or optional parameter that should be skipped. **/
```

Reply:
ok